lib/mycha.coffee

```diff
@@ -77,8 +77,7 @@ class Mycha
   # Parameters:
   # - mocha_args: Array of string arguments to provide to Mocha.
   call_mocha: (mocha_args, done) ->
-    childProcess = child.spawn path.resolve(__dirname, '../node_modules/mocha/bin/mocha'),
-                               mocha_args
+    childProcess = child.spawn 'mocha', mocha_args
```

Comment:
Does this work if `./node_modules/bin` is not in the path? Maybe check whether mocha is at `../node_modules/mocha/bin/mocha` and then use that path, otherwise fall back to the new solution here?